### PR TITLE
doc: add rolling.md

### PR DESCRIPTION
**Pull request by aslushnikov**
This docs has instructions on how to roll to Playwright driver

### CODE REVIEW

File: ROLLING.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Rolling Playwright-Python to the latest Playwright driver
+
+* checkout latest python from https://github.com/microsoft/playwright-python
+* make sure local python is 3.9+
```
Comment (mxschmitt) [inline ROLLING.md:4]:
```suggestion
* make sure local python is 3.9
```

File: ROLLING.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    * create virtual environment, if don't have one: `python3 -m venv env`
+* activate venv: `source env/bin/activate`
+* install all deps:
+     - `python -m pip install --upgrade pip`
+     - `pip install -r local-requirements.txt`
+     - `pip install -e .`
+* checkout `release-1.14` branch
+* change driver version in `setup.py`
+* download new driver: `rm -rf build/ dist/ driver/ && python setup.py bdist_wheel`
```
Comment (mxschmitt) [inline ROLLING.md:13]:
```suggestion
* download new driver: `python setup.py bdist_wheel`
```

File: ROLLING.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+* update API wrt new driver: `./scripts/update_api.sh`
```
Comment (mxschmitt) [inline ROLLING.md:14]:
```suggestion
* generate API: `./scripts/update_api.sh`
```

File: ROLLING.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+* run commit hooks (twice!): `pre-commit run --all-files`
```
Comment (mxschmitt) [inline ROLLING.md:15]:
```suggestion
```

File: ROLLING.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# Rolling Playwright-Python to the latest Playwright driver
+
+* checkout latest python from https://github.com/microsoft/playwright-python
+* make sure local python is 3.9+
+    * create virtual environment, if don't have one: `python3 -m venv env`
+* activate venv: `source env/bin/activate`
+* install all deps:
+     - `python -m pip install --upgrade pip`
+     - `pip install -r local-requirements.txt`
```
Comment (mxschmitt) [inline ROLLING.md:9]:
```suggestion
     - `pip install -r local-requirements.txt`
     - `pre-commit install`
```

File: ROLLING.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+# Rolling Playwright-Python to the latest Playwright driver
+
+* checkout latest python from https://github.com/microsoft/playwright-python
```
Comment (mxschmitt) [inline ROLLING.md:3]:
```suggestion
* checkout repo: `git clone https://github.com/microsoft/playwright-python`
```

**Review by mxschmitt — APPROVED** [review]

File: ROLLING.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+# Rolling Playwright-Python to the latest Playwright driver
+
+* checkout repo: `git clone https://github.com/microsoft/playwright-python`
+* make sure local python is 3.9
```
Comment (kumaraditya303) [inline ROLLING.md:4]:
```suggestion
* make sure python version is >=3.9
```

File: ROLLING.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+    * create virtual environment, if don't have one: `python3 -m venv env`
```
Comment (kumaraditya303) [inline ROLLING.md:5]:
```suggestion
    * create virtual environment, if don't have one: `python -m venv env`
```

File: ROLLING.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+* activate venv: `source env/bin/activate`
+* install all deps:
+     - `python -m pip install --upgrade pip`
+     - `pip install -r local-requirements.txt`
+     - `pre-commit install`
+     - `pip install -e .`
+* checkout `release-1.14` branch
```
Comment (kumaraditya303) [inline ROLLING.md:12]: Should be master, isn't it ?

Comment (mxschmitt) [reply ROLLING.md:12]:
```suggestion
```

**Pull request merged**